jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/UserTaskServiceImpl.java

```diff
@@ -700,7 +700,8 @@ public Long saveContentFromUser(Long taskId, String userId, Map<String, Object>
 
     @Override
     public Long saveContent(String deploymentId, Long taskId, Map<String, Object> values) {
-        return saveContent(deploymentId, taskId, null, values);
+        String userId = System.getProperty("org.jbpm.ht.admin.user", "Administrator");
```

Comment:
not sure why an admin user is required here to saveContent? shouldn't this user be fetched from `userGroupCallback` instead? can you please clarify?